CHANGELOG.md

```diff
@@ -0,0 +1,215 @@
+# Changelog:
+
+#### 2.12 (August 6, 2019)
```

Comment:
```suggestion
## 2.12
August 6, 2019
```
The date is kind of a nit, but I remember when I looked at the migration guildelines [here](https://jenkins.io/doc/developer/publishing/wiki-page/#reviewing-the-documentation), they do recommend putting date on the second line (see second bullet of number 2).